mempool/clist_mempool.go

```diff
@@ -598,6 +575,7 @@ func (mem *CListMempool) ReapMaxTxs(max int) types.Txs {
 }
 
 // Lock() must be help by the caller during execution.
+// TODO: always returns nil
```

Comment:
Well, it may return an error. Strictly speaking the function is void.

Reply:
Yes, what I wanted to say is that, because `Update`  always returns nil, we should remove the returning error. But I would leave that for another PR.